applications/dashboard/views/profile/edit.php

```diff
@@ -6,6 +6,13 @@
     echo $this->Form->errors();
     ?>
     <ul>
+        <?php
```

Comment:
This message will be seen by everyone including Admins. No one will be able to edit the username or email field when SSO is enabled. However, Admins can still edit users through dashboard. I think this is a lot less confusing to admins and users.